### ISSUE TITLE
checksum mismatch

### ISSUE THREAD

**Issue by conitrade-as** [BWalti/abaclient-chocolatey-package#2]
Seems that the package checksum is still not matching.
```
PS C:\WINDOWS\system32> choco install abaclient -y
Chocolatey v0.10.15
Installing the following packages:
abaclient
By installing you accept licenses for the packages.
Progress: Downloading abaclient 2.2.672... 100%

abaclient v2.2.672 [Approved]
abaclient package files install completed. Performing other installation steps.
Downloading abaclient
  from 'https://storage.googleapis.com/images.abasky.net/119676df089a90ab3c3004398806433938052586dcd9a6bcacfb0e3e4024f339/2.2.672/abaclient-2.2.672-de.msi'
Progress: 100% - Completed download of C:\ProgramData\chocolatey\lib\abaclient\tools\abaclient-2.2.672-de.msi (210.59 MB).
Download of abaclient-2.2.672-de.msi (210.59 MB) completed.
C:\ProgramData\chocolatey\lib\abaclient\tools\abaclient-2.2.672-de.msi
Error - hashes do not match. Actual value was 'DD5A74B3ABEE0BFFDAD411F23915DFDA86F82174DFB30A9B2C373E49301C2A73'.
ERROR: Checksum for 'C:\ProgramData\chocolatey\lib\abaclient\tools\abaclient-2.2.672-de.msi' did not meet 'CBEED96433DFD0058BE3ED080D60FC926D526D887889F691FBDA570A08373A23' for checksum type 'sha256'. Consider passing the actual checksums through with --checksum --checksum64 once you validate the checksums are appropriate. A less secure option is to pass --ignore-checksums if necessary.
The install of abaclient was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\abaclient\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - abaclient (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\abaclient\tools\chocolateyinstall.ps1'.
 See log for details.
```

This is how the checksum was calculated originally for the PR:
```
$ curl https://storage.googleapis.com/images.abasky.net/119676df089a90ab3c3004398806433938052586dcd9a6bcacfb0e3e4024f339/2.2.672/abaclient-2.2.672-de.msi | shasum -a 256
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  210M  100  210M    0     0  2755k      0  0:01:18  0:01:18 --:--:-- 1402k
dd5a74b3abee0bffdad411f23915dfda86f82174dfb30a9b2c373e49301c2a73  -
```

**Comment by BWalti** [comment]
Yeah I guess I made a stupid error when I created the package: the ps1 tries to fetch the install package matching the Windows UI language. But I only put in one CRC 😅 probably never tested it with other UI languages.. (shame on me)
I'd have to extend the ps1 with all three CRCs. As father to a 5 year old kid I'm not as regular at the PC so it may take some time. Sorry for any inconvenience!

**Comment by BWalti** [comment]
Had some time yesterday to try to build up a github actions pipeline..

tl;dr: new (non preview) package should be up as soon as all checks have been passed: 2.2.673.3

Thanks again for your PR!
